expression/builtin_regexp.go

```diff
@@ -1224,6 +1392,11 @@ func (re *builtinRegexpReplaceFuncSig) evalString(row chunk.Row) (string, bool,
 		re.once.Do(memorize) // Avoid data race
 	}
 
+	instructions, err := getInstructions([]byte(repl))
```

Comment:
I think if `repl` is constant, `getInstructions` only need to be called once.

Reply:
> I think if `repl` is constant, `getInstructions` only need to be called once.

okk